.github/ISSUE_TEMPLATE/Feature_request.md

```diff
@@ -7,6 +7,9 @@ assignees: ''
 
 ---
 
+**Related command**
+<!--- Please provide the related command with `az {command}`, azure-cli bot will analyse command and route to related team --->
```

Comment:
Use `analyze` (American English) instead of `analyse` (British English) as our most of our customers are in US.